spi/max7219_32x8_spi/README.adoc

```diff
@@ -0,0 +1,40 @@
+= Attaching a Max7219 driving an 32x8 LED display via SPI
+
+This example code shows how to interface the Raspberry Pi Pico to a Max7219 driving an 32x8 display. The particular device used is interfaced via SPI at 3.3v. This PCB uses 4 Max7219 devices, each driving an 8x8 LED matrix, giving the total of 32x8. These devices are designed to be cascaded together in this way, and SPI commands are passed down the line of devices as required.
+
+The Max7219 datasheet can be found here https://datasheets.maximintegrated.com/en/ds/MAX7219-MAX7221.pdf
+
+== Wiring information
+
+Wiring up the device requires 6 jumpers as follows:
+
+   * GPIO 16 (pin 21) MISO/spi0_rx-> DIN on max7219 board
+   * GPIO 17 (pin 22) Chip select -> CS on max7219 board
+   * GPIO 18 (pin 24) SCK/spi0_sclk -> CLK on max7219 board
+   * 5v (pin 40) -> VCC on max7219 board
+   * GND (pin 38)  -> GND on max7219 board
+
+The example here uses SPI port 0. Power is supplied from the 5V pin.
+
+[NOTE]
+======
+There are many different manufacturers who sell boards with the max7219. Whilst they all appear slightly different, they all have, at least, the same 5 pins required to power and communicate. When wiring up a board that is different to the one in the diagram, ensure you connect up as described in the previous paragraph.
```

Comment:
"to power and communicate" is probably better as "for power and communication" ?

"the diagram" ?? :wink: